src/index.js

```diff
@@ -84,6 +85,9 @@ class DropdownTreeSelect extends Component {
 
   handleClick = () => {
     this.setState(prevState => {
+      if (this.props.disabled) {
+        return { showDropdown: false }
+      }
```

Comment:
I think we can save some unnecessary render cycles if we simply add a no-op click handler for disabled

```
<a className={dropdownTriggerClassname} onClick={!this.props.disabled && this.handleClick}>
```